tests/std/tests/P1502R1_standard_library_header_units/importable_cxx_library_headers.json

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+// P1502R1_standard_library_header_units/test.cpp cites the definition of "importable C++ library headers".

```

Comment:
Though I also hate that original JSON does not allow comments, I don't think `.json` extension should be used for JSON+comments. Is there a standard extension for JSON with comments?

Reply:
There is `.json5` for json5.
it supports comments and few more features: https://json5.org/

Reply:
Hmm. It would be strange to have a `.json5` file but then parse it with Perl's built-in `JSON::PP`. However, I see that VSCode recognizes `.jsonc` as an extension. I can easily rename this test file, although `header-units.json` has already shipped under that name.